_config.yml

```diff
@@ -25,6 +25,7 @@ kramdown:
 
 include:
   - _static
+  - .well-known
```

Comment:
To be honest I am not sure what this include does but I think we should just copy the file on the deploy step:
https://github.com/apache/arrow-site/blob/aaa5896d63ca53208a098ff32d448afffe5c2c73/.github/workflows/deploy.yml#L80-L81
From the comments on this PR, I think this is where we copy it:
https://github.com/apache/arrow-site/pull/506

Reply:
The addition ensures that the `.well-known` directory and its contents is included in the generated site.
Is there a reason to prefer a shell `cp` command? I find it cleaner to rely as much as possible on configuration, rather than dedicated shell commands. Also, using `_config.yml` I could actually test my changes locally.

@kou What do you think?

Reply:
If this has been tested and works, I agree, this sounds like a better solution than manually `cp` files. We probably want to have a single way of doing this for the other files we are copying. 
Thanks for finding out!

Reply:
Ah, we should use `include` instead of raw `cp` for them too.
Sorry, I didn't notice this approach when I did it.